hardhat/hardhat.config.ts

```diff
@@ -23,7 +23,7 @@ task('accounts', 'Prints the list of accounts', async (args, hre) => {
   });
 });
 
-const config: HardhatUserConfig = {
+/*const config: HardhatUserConfig = {
   solidity: {
     compilers: [
```

Comment:
This won't work because we need to use forked mainnet changing it would break the deployment and building process.